var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -135,7 +135,7 @@ class Seacas(CMakePackage):
     variant("x11", default=True, description="Compile with X11")
 
     # ###################### Dependencies ##########################
-    depends_on("cmake@3.22:", when="@2023-10-24:", type="build")
+    depends_on("cmake@3.22:", when="@2023-11-27:", type="build")
```

Comment:
You can also revert these since the minimum requirement will hold for all future versions